modules/sdk/project-templates/src/test/java/com/liferay/project/templates/ProjectTemplateFilesTest.java

```diff
@@ -446,7 +446,8 @@ private void _testLiferayPluginPackageProperties(Path path)
 		_testPropertyValue(path, properties, "author", "${author}");
 		_testPropertyValue(path, properties, "change-log", "");
 		_testPropertyValue(path, properties, "licenses", "LGPL");
-		_testPropertyValue(path, properties, "liferay-versions", "7.0.0+");
+		_testPropertyContainsValue(
+			path, properties, "liferay-versions", "7.1.0+");
```

Comment:
@achaparro @jpince Before we merge, can we have this go back to `"7.0.0+, 7.1.0+"`. We want to maintain backwards compatibility for our users